compio-quic/Cargo.toml

```
@@ -0,0 +1,81 @@
+[package]
+name = "compio-quic"
+version = "0.1.0"
+description = "QUIC for compio"
+categories = ["asynchronous", "network-programming"]
+keywords = ["async", "net", "quic"]
+edition = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[dependencies]
+# Workspace dependencies
+compio-io = { workspace = true }
+compio-buf = { workspace = true }
+compio-log = { workspace = true }
+compio-net = { workspace = true }
+compio-runtime = { workspace = true, features = ["time"] }
+
+quinn-proto = "0.11.3"
+rustls = { workspace = true, features = ["ring"] }
+rustls-platform-verifier = { version = "0.3.3", optional = true }
+rustls-native-certs = { version = "0.7.1", optional = true }
+webpki-roots = { version = "0.26.3", optional = true }
+h3 = { version = "0.0.6", optional = true }
+
+# Utils
+bytes = "1.7.1"
+flume = { workspace = true }
+futures-util = { workspace = true }
+thiserror = "1.0.63"
+
+# Windows specific dependencies
+[target.'cfg(windows)'.dependencies]
+windows-sys = { workspace = true, features = ["Win32_Networking_WinSock"] }
+
+[target.'cfg(unix)'.dependencies]
+libc = { workspace = true }
+
+[dev-dependencies]
+compio-buf = { workspace = true, features = ["bytes"] }
+compio-dispatcher = { workspace = true }
+compio-driver = { workspace = true }
+compio-fs = { workspace = true }
+compio-macros = { workspace = true }
+compio-runtime = { workspace = true, features = ["criterion"] }
+
+rand = "0.8.5"
+rcgen = "0.13.1"
+socket2 = { workspace = true, features = ["all"] }
+tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
+
+criterion = { workspace = true, features = ["async_tokio"] }
+quinn = "0.11.3"
+tokio = { workspace = true, features = ["rt", "macros"] }
+http = "1.1.0"
+
+[features]
+default = ["h3"]
```

```suggestion
default = []
```

Don't add default features in sub-crates. Move it to the monocrate `compio`.